src/server/routes/signIn.ts

```diff
@@ -238,21 +238,29 @@ router.get(
   loginMiddleware,
   handleAsyncErrors(async (req: Request, res: ResponseWithRequestState) => {
     const state = res.locals;
-    const consents = await getUserConsentsForPage(
-      CONSENTS_POST_SIGN_IN_PAGE,
-      req.ip,
-      req.cookies.SC_GU_U,
-    );
-
-    const html = renderer('/signin/success', {
-      requestState: deepmerge(state, {
-        pageData: {
-          consents,
-        },
-      }),
-      pageTitle: 'Signed in',
-    });
-    res.type('html').send(html);
+    const { returnUrl } = state.pageData;
+    const { defaultReturnUri } = getConfiguration();
+    const redirectUrl = returnUrl || defaultReturnUri;
+
+    try {
```

Comment:
might be worth adding a comment to the e2e test about how these handle the case where the request to IDAPI was failing for an unknown reason